client/src/components/LandingPage/LandingPage.jsx

```diff
@@ -0,0 +1,80 @@
+import styles from './LandingPage.module.css';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import EmailIcon from '@mui/icons-material/Email';
+import { Link } from 'react-router-dom';
+import Features from './Features';
+import Hero from './Hero';
+import Navbar from '../Navbar/Navbar';
+import Footer from '../Footer/Footer';
+
+const LandingPage = () => (
+	<div className={styles.LandingPage}>
+		<>
+			<CssBaseline />
+			<Navbar></Navbar>
```

Comment:
We might want to pull the Navbar out of the Landing Page component and move it to App.jsx so that we have it on every page.

Reply:
Would this affect the login/sign up pages?